user_tools/src/spark_rapids_tools/cmdli/argprocessor.py

```diff
@@ -278,6 +291,13 @@ def define_invalid_arg_cases(self):
                 [ArgValueCase.VALUE_A, ArgValueCase.VALUE_A, ArgValueCase.IGNORE]
             ]
         }
+        self.rejected['Cluster By Properties Cannot go with OnPrem'] = {
+            'valid': False,
+            'callable': partial(self.validate_onprem_with_cluster_props),
+            'cases': [
+                [ArgValueCase.VALUE_A, ArgValueCase.UNDEFINED, ArgValueCase.UNDEFINED]
```

Comment:
For "Cluster By Properties", is the second value supposed to be `ArgValueCase.VALUE_B`, which is `ArgValueCase.UNDEFINED` for now?

Reply:
Thanks @cindyyuanjiang. Fixed it.

Reply:
Thanks @parthosa! Sorry I didn't notice this in the previous review, would it be more clear to specify this case is for "without eventlogs", both in the description and the error message in function `validate_onprem_with_cluster_props`?

Reply:
Thanks @cindyyuanjiang. Updated the method to reflect missing event logs. There was a redundant checking validation involved post platform assignment. I merged the validation so that we use the method `apply_arg_cases` to validate arguments before and after platform assignment.